reducer/aggregation/agg_root_span.cc

```diff
@@ -99,6 +104,7 @@ void AggRootSpan::update_node(
   // of the key.
   node_ref.modify().pod_name(pod_name);
 
+
```

Comment:
```suggestion
```

Reply:
Thanks for the review. Pushed a new patch removing the whitespace. Please review.